test/jdk/sun/security/provider/PolicyParser/ExtDirs.java

```diff
@@ -26,16 +26,15 @@
  * @bug 4215035
  * @summary standard extensions path is hard-coded in default
  *      system policy file
- * @run main/manual ExtDirs
+ * @run main ExtDirs
  */
 
 /*
- * Run this test manually with:
- * java    -Djava.security.manager \
- *         -Djava.ext.dirs=./ExtDirsA:./ExtDirsB \
- *         -Djava.security.policy==./ExtDirs.policy \
- *         -Djava.security.debug=parser \
- *         ExtDirs
+ * @test
+ * @bug 4215035
+ * @summary standard extensions path is hard-coded in default
+ *      system policy file
+ * @run main/othervm/policy=ExtDirs.policy/secure=default ExtDirs
```

Comment:
May be "/secure=default" is optional here.

Reply:
Yes, you are right and we can remove the /secure. According to the JTReg spec.
`If the /secure option is not used then the default security manager will be installed.`